data/projects.ini

```diff
@@ -164,6 +164,25 @@ git_url: https://github.com/boto/botocore.git
 description_short: Low-level, data-driven core of boto 3
 description_long: Low-level, data-driven core of boto 3
 
+[bulk-extractor]
+version: >=20200507
```

Comment:
Shoudn't this be 1.6.0 ?

Reply:
Yes, was using date format because there had not been any releases for some time. Updated to reflect the current version at the head.